src/agent/coverage/src/block/linux.rs

```diff
@@ -484,3 +474,15 @@ fn branch_target(inst: &iced_x86::Instruction) -> Option<(u64, bool)> {
         | FlowControl::XbeginXabortXend => None,
     }
 }
+
+fn continue_to_init_execve(tracer: &mut Ptracer) -> Result<Tracee> {
```

Comment:
This is copy-pasted in our `triage` crate. We could factor it out, but upstream should probably provide a similar method for the common `spawn()` use case.